tests/plugin/test_main.py

```diff
@@ -29,19 +29,27 @@ def test_multiple_paths_to_change(monkeypatch):
 
 Dummy docstring
 
+| Column name | Description | Type | Constraints |
+| ----------- | ----------- | ---- | ----------- |
+| patient_id | a column description | PseudoPatientId | Must have a value, must be unique. |
```

Comment:
thought: For me, if it's possible, I think that the formatting of the individual constraints *might* be easier to pick out if they are a bulleted list, e.g.

> * Must have a value
> * Must be unique

Because there may be a case where we have several constraints. Reading:

> Must be A, must be B, must not be C, must not be D…

can be a little disorientating, in my opinion, to pick out your position scanning the sentence. For two items, this is fine though.

But I don't think it matters for this PR: we can always refine the precise format later depending on:

* how it actually looks within MkDocs
* the text actually describing the constraints.